rhobs/alerting/data_plane/prometheus.gitops_argocd_app_reconcile_alerts.yaml

```diff
@@ -17,6 +17,7 @@ spec:
           Argo CD in namespace {{ $labels.namespace }}
           has failed to reconcile at least 99% of applications
           within 4 seconds over the past 30 minutes
+        namespace: '{{ if .Labels.dest_namespace }}{{ .Labels.dest_namespace }}{{ else if .Labels.namespace }}{{ .Labels.namespace }}{{ else }}""{{ end }}'
```

Comment:
Does  `dest_namespace` only exists some of the times?
Also, the expression depends on `namespace=gitops-service-argocd` so that label should always exist if the alert is firing.

I.e. I don't think we need the last else, and maybe neither the first.

If we keep the `else`, we need test cases that cover them.

Reply:
misunderstanding on my part, I don't know of situations when the `dest_namespace` are absent. refined, and added the required label to the time series.